core/contracts/types/entitlement_data_test.go

```diff
@@ -131,24 +131,6 @@ func TestConvertV1RuleDataToV2(t *testing.T) {
 				},
 			},
 		},
-		"Custom": {
```

Comment:
ISENTITLED checks are now v2 only, we don't need to test converting them to v1 rule data.